assets/javascripts/app/shortcuts.coffee

```diff
@@ -59,7 +62,7 @@ class app.Shortcuts
         @trigger 'escape'
         false
       when 32
-        if event.target.type is 'search' and (not @lastKeypress or @lastKeypress < Date.now() - 500)
+        if event.target.type is 'search' and @spaceScroll() and (not @lastKeypress or @lastKeypress < Date.now() - 1000)
```

Comment:
```suggestion
        if event.target.type is 'search' and @spaceScroll() and (not @lastKeypress or @lastKeypress < Date.now() - 500)
```
Since space scrolling can be toggled, I think we should stick with the original delay. Existing users are used to it and a 1 second delay feels surprisingly long when you have to wait.